docs/content/integrations/fivetran.mdx

```diff
@@ -70,7 +70,7 @@ fivetran_instance = fivetran_resource.configured(
 )
 ```
 
-For more information on the additional configuration options available for the Fivetran resource, see the [API reference](/\_apidocs/libraries/dagster-fivetran#dagster_fivetran.fivetran_resource).
+For more information on the additional configuration options available for the Fivetran resource, see the [API reference](/_apidocs/libraries/dagster-fivetran#dagster_fivetran.fivetran_resource).
```

Comment:
Odd that these linting errors aren't being caught by CI/CD 🤔 cc @benpankow @yuhan

Reply:
Looks like maybe there's a bit of disconnect between the lint run and the mdx-format run.